components/ContributorsGrid.js

```diff
@@ -221,6 +223,9 @@ ContributorsGrid.propTypes = {
 
   /** Collective id */
   collectiveId: PropTypes.number,
+
+  /* Reference to the FixedSizedGrid */
+  gridRef: PropTypes.oneOfType([PropTypes.func, PropTypes.shape({ current: PropTypes.any })]),
```

Comment:
This seems unrelated to the current PR (updating the Twitter logo). Can you maybe do an interactive rebase?

Reply:
@Betree I have made the changes